File: two_sum.java
public class Solution {
    public int[] twoSum(int[] nums, int target) {
        int sum =0;
        int[] arr ={0,0};
        for(int i=0;i<nums.length;i++){
            for(int j=i+1;j<nums.length;j++){
                sum = nums[i]+nums[j];
                if(sum == target){
                    arr[0]=i;
                    arr[1]=j;
                    break;
                }
            }
        }
       return arr; 
    }
}

File: two_sum.py
class Solution(object):
    def twoSum(self, numbers, target):
        """
        :type numbers: List[int]
        :type target: int
        :rtype: List[int]
        """
        start = 0
        end = len(numbers)-1
        while start<=end:
            if numbers[start]+numbers[end] > target:
                end = end-1
            elif numbers[start]+numbers[end] < target:
                start = start+1
            else: return start+1,end+1